### PR TITLE
added `create_calendar` to client, and assoc tests.

### PR DESCRIPTION
**Pull request by jstacoder**
i thought it would be nice having an easy way to create a new calendar from the client.  #54

### CODE REVIEW

File: pycronofy/tests/test_client.py [hunk 1]
```diff
@@ -152,6 +156,52 @@ def test_userinfo(client):
...
+        "profile_id": "%s",
+        "profile_name": "my-test-profile-name",
+        "calendar_id": "test-calendar-id",
+        "calendar_name": "%s",
+        "calendar_readonly": false,
+        "calendar_deleted": false,
+        "calendar_primary": false,
+        "permission_level": "unrestricted"
+        }}""" % (TEST_PROFILE_ID, TEST_CALENDAR_NAME),
+        status=200,
```
Comment (gshutler) [inline pycronofy/tests/test_client.py]: I may be missing something but this doesn't look like it is exercising the "ignore duplicate" path that the test name suggests.

I would expect this call to have a response status of 422, and would probably need to add a response for `GET /v1/calendars` based on my (non-Python dev!) read of the `except` block.

Comment (jstacoder) [reply pycronofy/tests/test_client.py]: yep, i totally meant to make a test for that case. all done.

**Comment by jstacoder** [comment]
Just wondering how this looks? It would be great if we could go back to using this? Over our fork.

**Comment by gshutler** [comment]
Thanks @jstacoder, this was released in `1.7.3`.

**Pull request merged**